_locales/en/messages.json

```diff
@@ -821,8 +815,8 @@
 	"settings_adblocker_list_3": {
 		"message": "Ads + Trackers + Annoyances"
 	},
-	"settings_purple_box": {
-		"message": "Purple Box"
+	"settings_tracker_tally": {
```

Comment:
Let's not change the key names. It helps preserve the translation history in transifex.

Reply:
Ah ok, sorry, I think you did mention this yesterday. Should I revert the change I made to the element id name `settings-show-purple-box` as well?

Reply:
You don't need to revert the html/css ids.  Just the i18n keys.